app/src/main/kotlin/com/wire/android/ui/authentication/create/common/CreateAccountBaseViewModel.kt

```diff
@@ -248,14 +284,14 @@ private fun RequestActivationCodeResult.toCodeError() = when (this) {
     RequestActivationCodeResult.Success -> CreateAccountCodeViewState.CodeError.None
 }
 
-private fun RegisterClientResult.toCodeError() = when (this) {
+private fun RegisterClientResult.Failure.toCodeError() = when (this) {
     RegisterClientResult.Failure.TooManyClients -> CreateAccountCodeViewState.CodeError.TooManyDevicesError
     RegisterClientResult.Failure.InvalidCredentials -> CreateAccountCodeViewState.CodeError.DialogError.InvalidEmailError
     is RegisterClientResult.Failure.Generic -> CreateAccountCodeViewState.CodeError.DialogError.GenericError(this.genericFailure)
-    is RegisterClientResult.Success -> CreateAccountCodeViewState.CodeError.None
+    //is RegisterClientResult.Success -> CreateAccountCodeViewState.CodeError.None
```

Comment:
We can just remove these comments then

Reply:
First, I wanted to verify with you if my changes are ok and will not break something

Reply:
I don't think it can break anything, in the case of success, we just navigate to another screen and don't allow to go back so we don't even need to clear the error I think 🤔